crates/fluvio-storage/src/replica.rs

```diff
@@ -721,11 +698,11 @@ mod tests {
             .await
             .expect("delete base dir");
 
-        let error = replica
+        let size = replica
             .get_partition_size()
             .await
-            .expect_err("error partition size");
-        assert_eq!(error, dataplane::ErrorCode::StorageError)
+            .expect("error partition size");
+        assert_eq!(size, 79);
```

Comment:
I had to change this test, removing the logs' directory does not lead to the `get_partition_size` method to throw an error anymore.
Instead, it returns the memory-stored size of active and previous segments. Even if these segments don't exist anymore.
What kind of behavior are we expecting here?

Reply:
I think this is correct.  total size = active_segment_size + prev_segment_size.  if that doesn't reflect deletion then it is separate bug.